03_launch_mgmt_cluster.sh

```diff
@@ -314,6 +313,6 @@ else
   fi
 fi
 
-launch_baremetal_operator
-apply_bm_hosts
-launch_cluster_api_provider_metal3
\ No newline at end of file
+launch_standalone_ironic
+launch_cluster_api_provider_metal3
```

Comment:
Can you run it before` apply_bm_hosts` function?

Reply:
I think so, because now BMO will be deployed in CAPM3 and we will apply bmh hosts after it?

Reply:
Ah true, for some reason I thought BMO deployment is in apply_bm_hosts. Sorry.